server/src/main/java/edu/jhu/server/Game.java

```diff
@@ -429,11 +427,11 @@ public void handleAccusationRequest(JSONObject request, Player player) {
       // this player's turn is now over
       final JSONObject endTurnRequest = new JSONObject();
       endTurnRequest.put(Constants.EVENT_TYPE, EventType.END_TURN_REQUEST);
-      handleEvent(endTurnRequest, player);
+      handleEndTurnRequest();
     }
   }
 
-  public void handleEndTurnRequest(JSONObject request) {
+  public void handleEndTurnRequest() {
```

Comment:
might be good to add player as an argument and add check if it is their turn like other handle* methods do